helix-front/angular.json

```diff
@@ -81,12 +81,16 @@
           }
         },
         "test": {
-          "builder": "@angular-devkit/build-angular:karma",
+          "builder": "@angular-builders/jest:run",
```

Comment:
I followed [this guide](https://dev.to/kiranmantha/integrating-jest-with-latest-angular-version-4h4m) to migrate, adjusting the paths to helix-front's existing paths along the way.